### PR TITLE
Add --capabilities flag to perun create-stack

### PR DESCRIPTION
**Pull request by piwowarc**
I added--capabilities flag to perun create-stack and fixed logging bug.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +140,12 @@ Example JSON template which describe S3 Bucket:
 If you want to destroy stack just type its name.
 Before you create stack you should validate it with perun :wink:.
 
+### Capabilities
+If your template includes resources that can affect permissions in your AWS account, 
+you must explicitly acknowledge its capabilities by adding `--capabilities=CAPABILITIES` flag.
+
+Valid values are `CAPABILITY_IAM` and `CAPABILITY_NAMED_IAM`.
```
Comment (afronski) [inline README.md]: Or both. ;) How to pass both?

File: README.md [hunk 2]
```diff
@@ -139,6 +140,12 @@ Example JSON template which describe S3 Bucket:
 If you want to destroy stack just type its name.
 Before you create stack you should validate it with perun :wink:.
 
+### Capabilities
```
Comment (afronski) [inline README.md]: Missing empty line after header.

**Comment by afronski** [comment]
@mwpolcik Can you verify if that fix is working fine for you?

**Comment by mwpolcik** [comment]
@afronski @piwowarc  Working as it should - thanks for the fix.

**Pull request merged**